### ISSUE TITLE
Why in soft_ labels function, the num of label should be subtracted by 1

### ISSUE THREAD

**Issue by daiyulejia** [yumeng5/RoSTER#3]
type_distrib = torch.zeros(input_ids.size(0), self.max_seq_length, self.num_labels-1).to(self.device

in this code,the -1 dim is num_labels-1 .is you delete the type"O" (other)?

thanks,

**Comment by yumeng5** [comment]
Yes, you are right. The `type_distrib` is the entity type prediction distribution (excluding the "O" label). Please refer to the paragraph "Implementation of the NER Head" in Appendix C of the paper.

Best,
Yu
